web/src/utils/dashboard/convertSQLData.ts

```
@@ -1800,7 +2180,8 @@ export const convertSQLData = async (
         }
       });
 
-      options.xAxis[0].type = "time";
+      options.xAxis.forEach((it: any) => (it.type = "time"));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)